.github/workflows/ci.yml

```diff
@@ -113,11 +116,19 @@ jobs:
       run: make crypto/relic/build
     - name: Run tests
       if: github.actor != 'bors[bot]'
-      run: make -C crypto cross-blst-test
+      run: |
+        if ! (grep -q -e '^flags.*\badx\b' /proc/cpuinfo) 2>/dev/null; then
+            export CGO_CFLAGS="-D__BLST_PORTABLE__"
+        fi
+        make -C crypto cross-blst-test
     - name: Run tests (Bors)
       if: github.actor == 'bors[bot]'
       uses: nick-invision/retry@v2
       with:
         timeout_minutes: 25
         max_attempts: 2
-        command: make -C crypto cross-blst-test
+        command: |
```

Comment:
As a general comment, can the `if` statement be run locally too and not only on CI ?

If yes, it would be cleaner to have a new Makefile target `crypto/cross-blst` that encapsulates all this. 
Eventually, when we make sure this cross-blst tests always pass, the new target would be called by the unit-tests and we can remove the blst CI job

Reply:
I captured this idea in https://github.com/onflow/flow-go/issues/1229
In theory, yes we could. In practice, ADX was introduced with Broadwell, and is accepted in most modern machines. I believe in GH this will fail on the Xeon E5-2673 v3, so I'm wary of over-optimizing for a niche case. The perf of CI post this change will tell.